test/fctGraph/seeds/connectedFCTGraphData.js

```diff
@@ -2581,7 +2581,7 @@ module.exports = {
   },
   {
     "id":"98e62d7c-a8d2-4490-be72-0bfaa96a4eaf",
-    "name":"WebReporter",
+    "name":"WebconnectedFCTGraphData",
```

Comment:
Why? :P The previous name was better!

Reply:
i have never seen this code 
```suggestion
    "name":"WebReporter",
```